src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -71,7 +66,10 @@ public String getContents() {
     public void toQuestion(Question question) {
         this.question = question;
     }
-
+    public DeleteHistory delete() {
+        this.deleted = true;
+        return new DeleteHistory(ContentType.ANSWER, id, writer, LocalDateTime.now());
+    }
```

Comment:
만약 삭제 시간과 관련된 로직이 존재하면 다음과 같이도 사용할 수 있답니다 😄 

```suggestion
    public DeleteHistory delete(LocalDateTime deletedAt) {
        this.deleted = true;
        return new DeleteHistory(ContentType.ANSWER, id, writer, deletedAt);
    }
```
참고만 해주세요~